lib/src/user.dart

```diff
@@ -40,6 +40,7 @@ class User {
   }
 
   late final ApiKeyClient _apiKeys = ApiKeyClient._(this);
+  late final FunctionsClient _functions = FunctionsClient._(this);
```

Comment:
```suggestion
  FunctionsClient? _functions
```

Reply:
So my suggestion is not worth it. The `late final` field is already lazy. Can we revert this suggestion